modules/codelite/tests/test_codelite_config.lua

```diff
@@ -57,11 +57,12 @@
 	end
 
 	function suite.OnProjectCfg_Includes()
+		sysincludedirs { "sysdir", "sysdir2/"}
```

Comment:
This test isn't for `sysincludedirs`, create a new test.

Reply:
I saw it as all include directives.
split anyway.